src/main/java/org/sagebionetworks/web/client/widget/table/v2/TableEntityWidget.java

```diff
@@ -277,6 +277,8 @@ private void configureActions() {
 			if (!canEdit) {
 				this.actionMenu.setDownloadActionsDisabledTooltipText("A draft version of a Dataset cannot be downloaded");
 			}
+		} else if(!entityBundle.getPermissions().getCanDownload()) {
+			this.actionMenu.setTableDownloadOptionsEnabled(false);

```

Comment:
Do we need to set tooltip text for this case?

Reply:
The tooltip text here is set in the setDownloadAcrionsEnabled function so this should be ok